packages/pluggableWidgets/datagrid-web/src/Datagrid.tsx

```diff
@@ -42,12 +40,24 @@ export default function Datagrid(props: DatagridContainerProps): ReactElement {
     }, [props.onConfigurationChange]);
 
     const customFiltersState = props.columns.map(() => useState<FilterFunction>());
```

Comment:
Adding more info from @tamelbrun :

I chose that line because of the useState<FilterFunction>(), which indicates that the initial state of the filters is always empty and not restored from the viewstate when you navigate back to the page. I just wanted to make sure that was on your radar

Reply:
To be addressed later -> https://mendix.atlassian.net/browse/WC-487

Reply:
👍🏻